.semaphore/semaphore.yml

```diff
@@ -344,3 +344,7 @@ blocks:
         - name: KSQL Time Concepts Tutorial tests
           commands:
             - make -C _includes/tutorials/time-concepts/ksql/code tutorial
+        
+        - name: KSQL Test ksqlDB concatenate columns tests
+          commands:
+            - make -C _includes/tutorials/ksql-concatenation-columns/ksql/code tutorial
```

Comment:
@bbejeck all file names should not include `ksql` because later an equivalent tutorial could be written for KStreams.  So `ksql-concatenation-columns` --> `concatenation-columns`, etc